### PR TITLE
Fix https://github.com/ubports/libertine/issues/33

### PR DESCRIPTION
**Pull request by arubislander**
After the transition from `PageStack` to `AdaptivePageLayout` for the Libertine configuration views some more fixes where needed:
`Page.pageStack.currentPage` is undefined for an `AdaptivePageLayout` so the `installPackage()` method on the `ContainerEditView` was no longer being called.

I added a property `homeView` to `DebianPackagePicker` and `SearchResultsView`, pass the instance of `ContainerEditView` (which has the id of _homeView_) to them and access the method via this property.

### CODE REVIEW

**Comment by dobey** [comment]
Hmm. I think it would be better instead to provide a common signal which the pages emit, and which is connected when creating those page components, rather than passing the home view in as a property, and then calling a method on it.

**Comment by arubislander** [comment]
Thanks for the feedback. Will look into it at the first opportunity.

**Comment by arubislander** [comment]
Have added a commit following your pointers and suggestion

File: qml/common/DebianPackagePicker.qml [hunk 1]
```diff
@@ -28,6 +28,7 @@ Page {
         title: i18n.tr("Available Debian Packages to Install")
     }
 
+    signal packageSelected( string packageName )
```
Comment (dobey) [inline qml/common/DebianPackagePicker.qml]: Please remove the extra whitespace inside the parentheses.

File: qml/common/DebianPackagePicker.qml [hunk 2]
```diff
@@ -45,7 +46,7 @@ Page {
 
         function install(fileName) {
             pageStack.removePages(debianPackagePicker)
-            pageStack.currentPage.installPackage(containerConfigList.getDownloadsLocation() + "/" + fileName)
+            debianPackagePicker.packageSelected( containerConfigList.getDownloadsLocation() + "/" + fileName )
```
Comment (dobey) [inline qml/common/DebianPackagePicker.qml]: Please remove the extra starting/ending whitespace inside parentheses.

File: qml/common/SearchResultsView.qml [hunk 3]
```diff
@@ -46,6 +45,7 @@ Page {
     property var currentContainer: null
 
     signal newSearch(string container)
+    signal packageSelected( string packageName )
```
Comment (dobey) [inline qml/common/SearchResultsView.qml]: Please remove the extra whitespace to keep consistent with the code.

**Review by dobey — CHANGES_REQUESTED** [review]

**Review by dobey — APPROVED** [review]
Looks OK now. Would be nice to have tests for this stuff too, but it looks like there are no tests for the QML at all yet, so I won't block your changes on that. Thanks!

**Pull request merged**